microsoft-edge/progressive-web-apps-chromium/index.md

```diff
@@ -59,57 +59,55 @@ With [Progressive Web Apps][MDNApps] (or simply *PWA*s) you do not have to decid
 
-When you build a *Progressive Web App* targeting web standard APIs, your application can be deployed across platforms and devices and take advantage of the device specific capabilities as available.  PWAs in Microsoft Edge (Chromium) are completely standards based from a web platform perspective and enable users to install the app directly from within the browser without the need for Store based deployment or registration.  Desktop PWAs are supported on any of the platforms Microsoft Edge (Chromium) is available, including Windows 7, Windows 10 and Mac OS.  Other benefits include:  
+When you build a *Progressive Web App* targeting web standard APIs, your application may be deployed across platforms and devices and take advantage of the device specific capabilities as available.  PWAs in Microsoft Edge (Chromium) are completely standards-based from a web platform perspective and enable users to install the app directly from within the browser without the need for Store-based deployment or registration.  Desktop PWAs are supported on any of the platforms Microsoft Edge \(Chromium\) is available, including Windows 7, Windows 10, and macOS.  Other benefits include:  
 
-*   Applications can be installed directly from within the browser via the "Install" icon in the navigation bar.  
+*   Applications may be installed directly from within the browser via the **Install** icon in the navigation bar.  
     
     ![Install application flyout and icon][ImageInstallPwa]  
     
-*   Applications can also be installed, run and managed from the "Settings"->"Apps" menu  
+*   Applications may also be installed, run and managed from the **Settings** > **Apps** menu  
     
     ![Application menu items under settings][ImageAppMenus]  
 
 *   Web Notifications are integrated into the Windows notification system
 *   Shared cookie store with the browser profile that installed the app
-*   Access to other browser features via the "..." menu including certificate validation, site permissions, tracking protection and browser extensions
-*   Full access to Microsoft [Edge DevTools][DevtoolsProgressiveWebApps] for debugging your app  
+*   Access to other browser features via the `...` menu including certificate validation, site permissions, tracking protection, and browser extensions
+*   Full access to [Microsoft Edge DevTools][DevtoolsProgressiveWebApps] for debugging your app  
 
 > [!IMPORTANT]
-> To tailor PWAs specifically for Windows 10 that call WinRT APIs from JavaScript, see the documentation specific to the EdgeHTML PWA features here:  [Progressive Web Apps (EdgeHTML)][PwaEdgehtmlIndex].  You can learn about testing your PWA on Windows 10, and distributing it in the Microsoft Store.  
+> To tailor PWAs specifically for Windows 10 that send WinRT API requests from JavaScript, see the [documentation specific to the EdgeHTML PWA features][PwaEdgehtmlIndex].  Learn  more about testing your PWA on Windows 10 and distributing it in the Microsoft Store.  
```

Comment:
"send WinRT API requests" > "make WinRT API requests"

Reply:
Updated. 
"To tailor PWAs specifically for Windows 10 that make WinRT API requests using JavaScript... "